src/styles/_catalog.scss

```diff
@@ -1,22 +1,43 @@
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  place-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(4, $card-width);
+
+  @media screen and (max-width: 1023.9px) {
```

Comment:
I suggest you to always start from mobile styles by default 
and use `min-width` to override it for larger screens

Reply:
sure i will, thank you!